From 4b4e18a38f4fbc88bc68601d179279f2a4f6b13f Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 10 Jan 2017 00:45:38 +0100
Subject: [PATCH 4/7] Do not use slashes in section ID attributes

These are not valid in DocBook, and we should avoid using an ID that
will need to be changed later on, so that the ID can be preserved.
---
 policy.sgml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 084472c..fb1f30f 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -7438,7 +7438,7 @@ rmdir /usr/local/share/emacs 2>/dev/null || true
       <sect id="sysvinit">
 	<heading>System run levels and <file>init.d</file> scripts</heading>
 
-	<sect1 id="/etc/init.d">
+	<sect1 id="etc-init.d">
 	  <heading>Introduction</heading>
 
 	  <p>
@@ -7833,7 +7833,7 @@ invoke-rc.d <var>package</var> <var>action</var>
             which contained scripts which were run once per machine
             boot. This has been deprecated in favour of links from
             <file>/etc/rcS.d</file> to files in <file>/etc/init.d</file> as
-            described in <ref id="/etc/init.d">.  Packages must not
+            described in <ref id="etc-init.d">.  Packages must not
             place files in <file>/etc/rc.boot</file>.
 	  </p>
 	</sect1>
-- 
2.11.0

